test/microbench/fam-api-mb/fam_region_spanning_atomic.cpp

```diff
@@ -0,0 +1,616 @@
+/*
```

Comment:
Can you add explanation of command line arguments and how to run somewhere.

Reply:
Done

Reply:
Done